src/Chapter3.hs

```diff
@@ -971,6 +1079,23 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data Day = Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday
+  deriving (Show, Eq, Enum, Bounded)
+
+isWeekend :: Day -> Bool
+isWeekend Saturday = True
+isWeekend Sunday = True
+isWeekend _ = False
+
+nextDay :: Day -> Day
+nextDay day = if day == maxBound then minBound else succ day
+
+daysToParty :: Day -> Int
+daysToParty Sunday = 5
+daysToParty Saturday = 6
+daysToParty day = fromEnum Friday - fromEnum day
```

Comment:
Nice solution with pattern-matching and usage of `fromEnum`! 👍🏻 You can write a solution even shorter if you use `mod` (and the fact of how it works for negative numbers)